### ISSUE TITLE
Stacks Proof-Of-Work Recognition Program - Goodkitty

### ISSUE THREAD

**Issue by Hero-Gamer** [stacksgov/decentralized-grants#27]

**Comment by thiskittyisgood** [comment]
**Milestone 2 Completed – September 30, 2024**

I’m pleased to announce that we’ve completed Milestone 2 of the project, focusing on the planning and preparation of the POWER Awards (Proof of Work, Recognition, and Excellence). These awards are designed to recognize individuals and projects that have made a significant impact on the Stacks ecosystem, highlighting contributions in the areas of creativity, technical development, and community leadership.

**POWER Awards Overview:**
The POWER Awards celebrate outstanding contributions in three categories:

- Creators: Artists, NFT creators, and content producers who have contributed to the ecosystem through creative works.
- Builders: Developers, app creators, and protocol designers who have built innovative tools, dApps, or technical infrastructure.
- Community Leaders: Ambassadors, educators, and organizers who have driven community growth and engagement within Stacks.

**Award Prizes:**

- Top 3 Winners (one per category): $425 each.
- Community Choice Award: $225 (voted by the community, any category).
- Special Recognition Award: Badge on Zero Authority, awarded to an exceptional project selected by the council.

**Judging Criteria:**
Submissions will be evaluated based on: 
- **Originality:** How innovative or unique the project is.
- **Impact on the Stacks Ecosystem:** How the project has contributed to growth or improvements.
- **Excellence:** Quality of execution or development, including creative, technical, or community excellence.
- ****Community Involvement:**** The level of engagement and support garnered from the community.

**Submission Process:**
The submission process involves:
- **Proof of Work Submission:** Entrants must submit documentation showcasing their contributions and achievements.
- **Submit your project** starting October 3rd via https://www.stackslink.xyz/power-awards.
- **Council Verification:** Entrants must be verified by two members of the council to ensure authenticity and impact.
- Attendance Requirement: Entrants must attend at least two spaces in October and present their work during the review sessions.

**Timeline & Key Dates:**
**Submissions Open:** October 3rd, 2024. (Announcement https://twitter.com/i/spaces/1djGXrXNjYzxZ
**Submission Form**: https://blocksurvey.io/project-submission-survey-NOajNX5ETNuQvmW.qhGAYA?v=o
**Submission Reviews & Mentorship:** October 10th, 2024 https://twitter.com/i/spaces/1dRKZdrwjZoJB and October 17th, 2024: https://twitter.com/i/spaces/1rmxPoRWaWVJN
**Community Voting Opens:** October 18th, 2024 (via https://www.stackslink.xyz/power-awards).
**Finalists Announced:** October 24th, 2024. https://twitter.com/i/spaces/1yoKMyrbvYnKQ
**Community Voting Closes**: October 28th, 2024.
**Winners Announced**: October 31st, 2024. https://twitter.com/i/spaces/1gqGvNdLvbwGB

**Promotional Strategy:**
- A comprehensive promotional campaign has been designed, including daily social media updates, spaces, and X threads to spotlight entries.
- A media kit will be provided to finalists and winners for use in promoting their achievements.

**Next Steps:**
- Finalize preparations for the voting platform and logistics for the award ceremony.
- Engage with the community to encourage submissions and voting participation.
- We’re excited to move forward with this important initiative and look forward to recognizing the incredible talent within the Stacks ecosystem.

**Comment by Hero-Gamer** [comment]
Yep that's awesome! Consider milestone 1 and 2 completed! They are $0 milestones so nothing for Finance to do here. 
Look forward to Milestone 3!

**Comment by melaniecarstens** [comment]
I'm so excited to confirm that the winners of the Proof of Work Retroactive Grant Funding have been announced.
https://x.com/StacksLink/status/1857314004919550104
https://x.com/thiskittyisgood/status/1857307599286263841

Please make payments as follows:

**Builder: Parrot Radio $400**
The Pandemonium  [brett@gamma.io](mailto:brett@gamma.io)
SP467VFDYHV185JSQ98V9VTA7JS3PFJ3DM8PXD20

**Creator: Felix $400**
Vini [me@vicnicius.com](mailto:me@vicnicius.com)
SP3RDC4C9B0A2FG8B7DQ9MBTFPYQZNDAVCBME8Q41

**Community Leader: DeOrganised Media $400**
PeaceLoveMusic aka Steve [perrinoproperties@gmail.com](mailto:perrinoproperties@gmail.com)
SP2Z2CBMGWB9MQZAF5Z8X56KS69XRV3SJF4WKJ7J9

**Community Choice Award: Web3Binta $200**
Aminu Binta [binkash69@gmail.com](mailto:binkash69@gmail.com)
SPBWF76FHRNA9C1A6ZZ896B3XRRK5TGGW7X9A55A

**Please note:**  The initial grant for this proposal was $1540 and the grantee payouts total $1400.
I would like to use the un-used balance of $140 to pay a designer to create the Zero Authority Badge.


**Summary of all grant applicants:** https://docs.google.com/document/d/1kgClUyXZj2aYrU5NpYfkZ308qIeQ0YWF8RUdLi0mKQw/edit?usp=sharing

**Voting:**
https://docs.google.com/spreadsheets/d/11-d1GMay6e-2SQeqhC_WOFS_0Mx1tn1RNelXpc-Ax68/edit?usp=sharing

**Comment by melaniecarstens** [comment]
Hi

Winners have all been paid.

https://explorer.hiro.so/txid/0xd2ee0d2faa69c9c26c97af0d54d8a06597467265890af4a08d40e2661729f326?chain=mainnet

https://explorer.hiro.so/txid/0x6c022f697139b39fb227d4aef7b1ff0c97519a9186b206627c8a26b804db9f0e?chain=mainnet

https://explorer.hiro.so/txid/0xd7abdd17017180bfc2c1322d3fc630c230be190972632c6f46a0fcf269f1f566?chain=mainnet

https://explorer.hiro.so/txid/0x4c6f7a1e82958e57a3e31e54619c1f228243ce702369f6fbab9272bcba01a34a?chain=mainnet

**Comment by Hero-Gamer** [comment]
Goodkitty appreciate all the documentations for transparency for the future Stackers looking back in history!
